src/components/AutocompleteFilter.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import { Autocomplete, IconButton, OutlinedInput } from '@mui/material';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+import SearchIcon from '@mui/icons-material/Search';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+/* eslint-disable react/jsx-props-no-spreading */
+export default function AutocompleteFilter(props) {
```

Comment:
Make this a TSX file while you're at it. You can do spread props using [this trick](https://stackoverflow.com/a/58201122/4811506)